src/about.njk

```diff
@@ -26,10 +27,6 @@ meta:
         - https://twitter.com/flowfuseinc
         - https://github.com/FlowFuse
         - https://www.youtube.com/channel/UCbBzP8NZbv3WDtlt4UouA-g
-eleventyComputed:
```

Comment:
Why is this removed?

Reply:
I reverted it, what's the difference between ? 
```
meta:
        description:......
```

and 
````
meta:
        organization:
                            description:......
````